jetstream/analysis.py

```diff
@@ -44,6 +45,7 @@ class Analysis:
     project: str
     dataset: str
     config: AnalysisConfiguration
+    log_config: Optional[LogConfiguration] = None
```

Comment:
We need to pull in the log configuration so it can be passed into the plugin and used for setting up the loggers